.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+env:
+  - POSTGRES_DB=test_db POSTGRES_USER=test_user POSTGRES_PASSWORD=test_password
+python:
+  - "3.6"
```

Comment:
We can just add "2.7" in a new line here to run against both versions of python.

I can see how the makefile-managed docker containers work on travis.  I hesitated doing that because I assumed deviating from The Way you use travis would probably lead me down some sad rabbit holes.  I'll play with this and see how it works.